freeipa.spec.in

```diff
@@ -1142,7 +1142,7 @@ fi
 %{_usr}/share/ipa/updates/*
 %dir %{_localstatedir}/lib/ipa
 %attr(700,root,root) %dir %{_localstatedir}/lib/ipa/backup
-%attr(700,root,root) %dir %{_localstatedir}/lib/ipa/gssproxy
+%ghost %dir %{_localstatedir}/lib/ipa/gssproxy
```

Comment:
Please add owner and mode to the %ghost. rpm -V recently changed and compains if a ghosted file has no owner/mode.

Reply:
it is the problem, because the owner of this directory can be either `root` or `_gssproxy`.

Reply:
`/var/lib/ipa/gssproxy` dir is created on installation, an owner and permissions are fixed on upgrade.

Reply:
Yeah but rpm is more picky about %ghost, see https://github.com/freeipa/freeipa/pull/3144

Reply:
Since the spec file is for Fedora and RHEL, which utilize `root` for `gssproxy`, I add it as user and group. Fixed.